frontend/src/components/swiper/CoverFlowCard.jsx

```diff
@@ -0,0 +1,35 @@
+// Review how to make the cards wider
+
+const CoverFlowCard = ({data}) => {
+    return(
+      <div className="max-w-sm rounded overflow-hidden shadow-lg bg-white h-[20rem] w-[120rem] bg-center bg-cover">
+        <img className="h-[10rem] w-full" src={data.imageUrl} alt="https://www.shutterstock.com/image-vector/default-ui-image-placeholder-wireframes-600nw-1037719192.jpg" />
+        <div className="px-3 py-4 flex flex-col justify-center items-center">
+          <div className="font-bold text-base mb-2 mx-[1.3rem]">{data.name}</div>
+          <p className="text-gray-700 text-sm mx-[1.1rem]">
+            📍{data.location}
+          </p>
```

Comment:
Is this your code? I would be very careful about directly copy pasting here. If you do, do cite them. And also note that, you will be evaluated on how much code YOU write.

Reply:
Yes, this part is mine